models/census.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var censusSchema = mongoose.Schema({
+    'type': String,
+    'activity_year': String,
+    'data': [{
```

Comment:
Just curious, why do we need to store the rest of this information in a 'data' object?